build_and_test.sh

```diff
@@ -21,89 +24,52 @@ fi
 echo "==========================================================="
 echo "Test OpenFAM with cis-rpc-meta-direct-mem-rpc configuration"
 echo "==========================================================="
-export OPENFAM_ROOT="$MEMSRV_BUILD_DIR/test/config_files/config-cis-rpc-meta-direct-mem-rpc/"
-source setup.sh --cisserver=127.0.0.1  --memserver=127.0.0.1
-echo $OPENFAM_ROOT
-make unit-test
-if [[ $? > 0 ]]
-then
-        echo "One or more unit-test failed while running with cis-rpc-meta-direct-mem-rpc configuration. exit..."
-        exit 1
-fi
-make reg-test
+CONFIG_OUT_DIR=$MEMSRV_BUILD_DIR/test/config_files/config-cis-rpc-meta-direct-mem-rpc
+python3 $SCRIPT_DIR/run_test.py --launcher $LAUNCHER -n 1 --cisinterface rpc --cisrpcaddr 127.0.0.1:8787 --memserverinterface rpc --metaserverinterface direct --memserverlist 0:127.0.0.1:8788 $CONFIG_IN_DIR $CONFIG_OUT_DIR $MEMSRV_BUILD_DIR
```

Comment:
I think it is better to have an option in run_test.py to read all the command line arguments from a file. 
So that we need not bother to write a long list of command-line arguments.